tests/testthat/test-age_calculate.R

```diff
@@ -0,0 +1,77 @@
+test_that("Age is calculated correctly for all units", {
+  expect_equal(age_calculate(as.Date("2021-02-01"), as.Date("2022-02-21"),
+                             units = "years", date_class = "period",
+                             round_down = TRUE), 1)
+  expect_equal(age_calculate(as.Date("2021-02-01"), as.Date("2022-02-21"),
+                             units = "months", date_class = "period",
+                             round_down = TRUE), 12)
+  expect_equal(age_calculate(as.Date("2021-02-01"), as.Date("2022-02-21"),
+                             units = "weeks", date_class = "period",
+                             round_down = TRUE), 55)
+  expect_equal(age_calculate(as.Date("2021-02-01"), as.Date("2022-02-21"),
+                             units = "days", date_class = "period",
+                             round_down = TRUE), 385)
+  expect_equal(age_calculate(as.Date("2021-02-01"), as.Date("2022-02-21"),
+                             units = "hours", date_class = "period",
+                             round_down = TRUE), 9240)
+  expect_equal(age_calculate(as.Date("2021-02-01"), as.Date("2022-02-21"),
+                             units = "minutes", date_class = "period",
+                             round_down = TRUE), 554400)
+  expect_equal(age_calculate(as.Date("2021-02-01"), as.Date("2022-02-21"),
+                             units = "seconds", date_class = "period",
+                             round_down = TRUE), 33264000)
+})
```

Comment:
- [ ]  Add some tests for the other `date_class` option.
- [ ] Add some tests on the return type.
- [ ] Add some test for `round_down = FALSE` (`round_down = TRUE` isn't required here as it's the default.
- [ ] Add a test for start date after end date (i.e. negative age) - Could possibly include an error or warning for this? Maybe also warn for very large (>130) ages?